docs/help/diffusion.md

```diff
@@ -0,0 +1,18 @@
+Taking the time to understand the diffusion process will help you to understand how to more effectively use InvokeAI.
+
+There are two main spaces Stable Diffusion works in: image space and latent space.
+
+Image space represents images in pixel form that you look at. Latent space represents compressed inputs. It’s in latent space that Stable Diffusion processes images. A VAE (Variational Auto Encoder) is responsible for compressing and encoding inputs into latent space, as well as decoding outputs back into image space.
+
+When you generate an image using text-to-image, multiple steps occur in latent space:
+1. Random noise is generated at the chosen height and width. The noise’s characteristics are dictated by the chosen (or not chosen) seed. This noise tensor is passed into latent space. We’ll call this noise A.
```

Comment:
Unclear what you mean by “(or not chosen)”. Do you mean “(or randomly generated)”?